### PR TITLE
Add OpenFaaS Clear Linux based python3 template

### PR DESCRIPTION
**Pull request by qzheng527**
Signed-off-by: Qi Zheng <qi.zheng@intel.com>

### CODE REVIEW

File: FaaS/OpenFaaS/template/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    │   ├── __init__.py
+    │   └── requirements.txt
+    ├── hello-openfaas.yml
+    └── template
+
+
+    *  Put the required python packages in the "requirements.txt".
+    For example,
+
+    `echo "redis" >> hello-openfaas/requirements.txt`
```
Comment (ZhongbaoShi) [inline FaaS/OpenFaaS/template/README.md]: It seems these optional files (empty file) also submit together. Is there more better way handle this case? @bryteise

Comment (bryteise) [reply FaaS/OpenFaaS/template/README.md]: Sorry I had missed this. I don't think I quite understand your question though.

Comment (ZhongbaoShi) [reply FaaS/OpenFaaS/template/README.md]: Oh, I meaning 5 empty configuration file is submitting with other changes together. Such as, requirements.txt; bundles.txt .... Not sure if this is better solution? or if you have better idea?

Comment (qzheng527) [reply FaaS/OpenFaaS/template/README.md]: @ZhongbaoShi I think the empty files are fine. Users who use  the template can add what they want into the requirements.txt or bundles.txt.

Comment (bryteise) [reply FaaS/OpenFaaS/template/README.md]: I agree with @qzheng527, it isn't really a problem I don't think.

**Pull request merged**